src/middlewared/middlewared/etc_files/motd

```diff
@@ -1,9 +1,10 @@
 <%
+	from datetime import datetime as dt
 	motd = middleware.call_sync('system.advanced.config')['motd']
 %>\
 FreeBSD ?.?.?  (UNKNOWN)
 
-	FreeNAS (c) 2009-2019, The FreeNAS Development Team
+	FreeNAS (c) 2009-${dt.today().year}, The FreeNAS Development Team
```

Comment:
We dont want it to be dynamic.

We dont want software that was released in 2010 with 2020 copyright.